lib/commands.js

```diff
@@ -55,13 +55,50 @@ var commands = module.exports.commands = [
   , 'walletLock'
   , 'walletPassphrase'
   , 'walletPassphraseChange'
+  ,
+  'checkWallet',
```

Comment:
Seems odd that formatting is inconsistent with previous entries, could you use a consistent style and restore alphabetic ordering of the list?

Reply:
jeah, that looked kinda ugly

my recent commits fixed that